config.go

```diff
@@ -72,7 +72,7 @@ func readConfigFile(config interface{}, fileName string) error {
 	return nil
 }
 
-func (c menderConfig) GetUpdaterConfig() httpsClientConfig {
+func (c menderConfig) GetHttpConfig() httpsClientConfig {
```

Comment:
I would argue that `GetHttpConfig` is more natural. Changing it to `GetHttpsConfig` and returning a configuration without SSL related info would look awkward.